### PR TITLE
Version 0.8.1

### PR DESCRIPTION
**Pull request by russhwolf**

### CODE REVIEW

File: buildSrc/build.gradle.kts [hunk 1]
```diff
@@ -26,7 +26,7 @@ repositories {
 
 dependencies {
     implementation(kotlin("gradle-plugin"))
-    implementation("com.android.tools.build:gradle:4.1.2")
+    implementation("com.android.tools.build:gradle:4.2.2")
```
Comment (vanniktech) [inline buildSrc/build.gradle.kts:29]: Don't want to jump to 7.0.2 yet?

File: buildSrc/src/main/kotlin/Versions.kt [hunk 2]
```diff
@@ -17,22 +17,22 @@
...
-    const val coroutinesNativeMt = "1.5.1-native-mt"
+    const val coroutines = "1.5.2"
+    const val coroutinesNativeMt = "1.5.2-native-mt"
     const val junit = "4.13.2"
     const val robolectric = "4.6.1"
-    const val serializationPlugin = "1.5.30"
-    const val serializationRuntime = "1.2.2"
-    const val turbine = "0.6.0"
+    const val serializationPlugin = "1.5.31"
+    const val serializationRuntime = "1.3.0-RC"
```
Comment (vanniktech) [inline buildSrc/src/main/kotlin/Versions.kt:36]: In the changelog it's saying 1.2.2, is this change intentional?

**Review by vanniktech — APPROVED** [review]

File: buildSrc/build.gradle.kts (hunk 1 above)
Comment (russhwolf) [reply buildSrc/build.gradle.kts:29]: No, I use Intellij more than Android Studio for this and it's not 7.0.x-compatible yet

File: buildSrc/src/main/kotlin/Versions.kt (hunk 2 above)
Comment (russhwolf) [reply buildSrc/src/main/kotlin/Versions.kt:36]: That's a changelog typo. Thanks. Will fix.
